hashes/src/util.rs

```diff
@@ -3,28 +3,32 @@
 #[macro_export]
 /// Adds hexadecimal formatting implementation of a trait `$imp` to a given type `$ty`.
 macro_rules! hex_fmt_impl(
-    ($reverse:expr, $ty:ident) => (
-        $crate::hex_fmt_impl!($reverse, $ty, );
+    ($reverse:expr, $len:expr, $ty:ident) => (
+        $crate::hex_fmt_impl!($reverse, $len, $ty, );
     );
-    ($reverse:expr, $ty:ident, $($gen:ident: $gent:ident),*) => (
+    ($reverse:expr, $len:expr, $ty:ident, $($gen:ident: $gent:ident),*) => (
         impl<$($gen: $gent),*> $crate::_export::_core::fmt::LowerHex for $ty<$($gen),*> {
             #[inline]
             fn fmt(&self, f: &mut $crate::_export::_core::fmt::Formatter) -> $crate::_export::_core::fmt::Result {
+                use $crate::Hash as _;
```

Comment:
Is `_` necessary here?

Reply:
The `_` is a useful idiom which indicates "I'm importing this trait to use its methods but not to actually refer to it".

Reply:
Hey @sanket1729 we are trying a new thing were we don't merge PRs unless all conversations are resolved. Can you resolve this if you agree its resolved please.

Reply:
@tcharding, feel free to resolve _my_ conversations in future if that is blocking merging any PR. There will be instances where I will be MIA for weeks. 
I waive my right to resolve conversations myself and all maintainer can resolve them on my behalf.